### PR TITLE
add the user-agent header

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

**Review by Xiaochao8 — APPROVED** [review]

File: g11n-ws/vip-manager-l10n-agent/src/main/resources/application.properties [hunk 1]
```diff
@@ -24,6 +24,8 @@ source.sync.s3.startTime=2020-01-01 13:45:11
 remote.source.schedule.cron=0 0/5 * * * ?
 #csp l10 site addess
 remote.l10n.base.url=https://localhost:8088
+#define the user-agent name, default is NoAgent
+remote.l10n.base.user-agent=NoAgent
```
Comment (linr211) [inline g11n-ws/vip-manager-l10n-agent/src/main/resources/application.properties:28]: it should have an initial value(e.g sgtn_l10n_agent), having a harcoded 'NoAgent' seems meaningless.
so if there's no 'remote.l10n.base.user-agent' or it's empty, it doesn't need to add the header.

File: g11n-ws/vip-manager-l10n-agent/src/main/java/com/vmware/l10agent/base/PropertyContantKeys.java [hunk 2]
```diff
@@ -24,6 +24,7 @@ public interface PropertyContantKeys {
 	public final static String PRODUCT_NAME = "productName";
 	public final static String VERSION = "version";
 	public final static String LOCAL = "local";
+	public final static String NO_AGENT = "NoAgent";
```
Comment (linr211) [inline g11n-ws/vip-manager-l10n-agent/src/main/java/com/vmware/l10agent/base/PropertyContantKeys.java:27]: This variable is not required if there's an initial value in the configuration.

File: g11n-ws/vip-manager-l10n-agent/src/main/resources/application.properties (hunk 1 above)
Comment (tigershi) [reply g11n-ws/vip-manager-l10n-agent/src/main/resources/application.properties:28]: done

**Review by linr211 — APPROVED** [review]

**Review by jin-jasper — APPROVED** [review]

File: g11n-ws/vip-manager-l10n-agent/src/main/resources/application.properties [hunk 3]
```diff
@@ -24,6 +24,8 @@ source.sync.s3.startTime=2020-01-01 13:45:11
 remote.source.schedule.cron=0 0/5 * * * ?
 #csp l10 site addess
 remote.l10n.base.url=https://localhost:8088
+#define the user-agent name, default is NoAgent
```
Comment (Xiaochao8) [inline g11n-ws/vip-manager-l10n-agent/src/main/resources/application.properties:27]: remove this comment

**Pull request merged**